master/reference/cni-plugin/configuration.md

```diff
@@ -335,3 +335,36 @@ There are two annotations to request a specific IP address:
 > - You can only specify one IPv4/IPv6 or one IPv4 and one IPv6 address with these annotations.
 > - When `ipAddrs` or `ipAddrsNoIpam` is used with `ipv4pools` or `ipv6pools`, `ipAddrs` / `ipAddrsNoIpam` take priority.
 {: .alert .alert-info}
+
+#### Requesting a floating IP
+
+You can request a floating IP address for a pod through [Kubernetes annotations](https://kubernetes.io/docs/user-guide/annotations/) with {{site.prodname}}.
```

Comment:
A little late to the convo on this, but should we mention that the floating IP must be part of a defined calico pool for it to be advertised properly?

Reply:
Yes, that is a good point. We should update the doc to mention that.